docs/api-integration/tlm_event_socket.md

```diff
@@ -0,0 +1,308 @@
+A `message` is the link between a `user` and an `event`.  
+A `message` will be created for each `users` to which Tracim should send the `event`.
+
+A `TLM` is the action of sending the `message` to the frontend, to the logged used.
+
+**Example:**  
+When a content is created in a space, 1 `event` is created.  
+If the space contains 10 users, 10 `messages` are created.  
+If 3 users are currently logged in to Tracim, 3 `TLM` are sent.  
+
+TLM are sent through **Server Sent Event (SSE)** and will trigger the update of the frontend's interface.
+
+## Connection
+
+Using **Server Sent Event (SSE)**, connect to:
+```
+/api/users/<user_id>/live_messages
+```
+Where **<user_id>** is the id of the logged user.
+
+**Example:**
+```js
+const TLMConnection = new EventSource(
+  `/api/users/${userId}/live_messages`,
+  { withCredentials: true }
+)
+```
+
+## Protocol
+
+TLM are read only. SSE are one-way connection.
+
+TLM body is in JSON.
+
+TLM object properties are:  
+**event_id**<**int**>: Id of the event in database  
+**event_type**<**string**>: Identification of the event. See [Identification: Event type](#identification-event-type)  
+**created**<**string**>: ISO datetime of the creation of the event  
+**read**<**string** | **null**>: ISO datetime of when is the user read the TLM. Usually null  
+**fields**<**object**>: Data of the TLM. See [Data: Fields](#data-fields)  
+
+**Example:**
+```json
+{
+  "event_id": 99,
+  "event_type": "content.created.thread",
+  "created": "2025-03-27T13:51:25Z",
+  "read": null,
+  "fields": {}
+}
+```
+
+### Identification: Event type
+
+To identify the action that has happened, a TLM has an `Event type`.  
+`Event type` are made from 3 parts each separated by a dot ".":
+- Entity type
+- Core event type
+- Sub type (optional)
+
+**Entity type** is the kind of data object the TLM is about.  
+List of entity type: `user`, `workspace`, `workspace_member`, `content`, `mention`, `reaction`, `workspace_subscription`, `tag`, `content_tag`, `user_call`
+
+**Core event type** is the kind of action the TLM is about.  
+List of core event type: `copied`, `created`, `deleted`, `undeleted`, `modified`, `moved`
+
+**Sub type** is optional and is a sub list of entity type when the entity type groups several data objects  
+List of sub type: only the entity type `content` currently has sub type and they are: `thread`, `html-document`, `folder`, `comment`, `kanban`, `logbook`, `todo`
+
+**Examples**:  
+Event type of TLM sent when a user change his avatar: `user.modified`  
+Event type of TLM sent when a user join a space: `workspace_member.created`  
+Event type of TLM sent when a content thread has been created: `content.created.thread`  
+
+### Data: Fields
+
+Fields are the data related to the TLM. See [Fields table](#fields-table).
+
+Each entry in fields is a subset of the corresponding HTTP API structure.
+
+⚠️ "content" field structure vary depending on the type of content (see below).
```

Comment:
Maybe use this emoji is better than `*note` (maybe use this each time we need to make a focus).

Reply:
I think they serve 2 different purpose
*note is an added information
the emoji is here to raise awareness on some behavior that can be misleading